cardano-cli/src/Cardano/CLI/Shelley/Output.hs

```diff
@@ -31,12 +32,14 @@ data QueryTipLocalState = QueryTipLocalState
   { era :: AnyCardanoEra
   , eraHistory :: EraHistory CardanoMode
   , mSystemStart :: Maybe SystemStart
+  , epochInfo :: EpochInfo (Either TransactionValidityIntervalError)
   }
 
 data QueryTipLocalStateOutput = QueryTipLocalStateOutput
   { mEra :: Maybe AnyCardanoEra
   , mEpoch :: Maybe EpochNo
   , mSyncProgress :: Maybe Text
+  , mSystemStartOut :: Maybe SystemStart
```

Comment:
Was there a new requirement to output the system start?

Reply:
Needed here: https://github.com/input-output-hk/cardano-node/pull/3127/files#diff-ce34c52a5affc68370e5db5016ace60dd627d0c9e7b495191bc52bf5ccc7bd65R224